djangocms_versioning/plugin_rendering.py

```diff
@@ -0,0 +1,54 @@
+from functools import lru_cache
+
+from django.apps import apps
+
+from cms.plugin_rendering import ContentRenderer
+
+from .constants import DRAFT, PUBLISHED
+
+
+def get_versionable_for_grouper(model):
+    versioning_extension = apps.get_app_config('djangocms_versioning').cms_extension
+    if versioning_extension.is_grouper_model_versioned(model):
+        return versioning_extension.versionables_by_grouper[model]
+
+
+@lru_cache()
```

Comment:
woa. toolbar is a per request instance, you don't want this cached at the process level